views/index.erb

```diff
@@ -0,0 +1,5 @@
+<ul>
+  <% @messages.each do |messages| %>
```

Comment:
Great layout, you may want to name the iterator to the singular of messages.
E.g. 
@messages.each do |message|
message
end